src/azure-cli/azure/cli/command_modules/network/tests/latest/recordings/test_network_express_route_port.yaml

```diff
@@ -342,7 +342,7 @@ interactions:
       User-Agent:
       - AZURECLI/2.26.0 azsdk-python-azure-mgmt-keyvault/9.0.0 Python/3.8.10 (Windows-10-10.0.19043-SP0)
     method: GET
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/cli_test_express_route_port000001/providers/Microsoft.KeyVault/vaults/test-er-port-kv000002?api-version=2021-06-01-preview
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/cli_test_express_route_port000001/providers/Microsoft.KeyVault/vaults/test-er-port-kv000002?api-version=2022-07-01
```

Comment:
I can't rerun this test successfully in live mode. So I just replace keyvault's API version to unblock CI.

@necusjz will fix this test in separate PR